esp32_ulp/opcodes.py

```diff
@@ -423,7 +423,7 @@ def i_move(reg_dest, reg_imm_src):
     if src.type == REG:
         _alu_reg.dreg = dest
         _alu_reg.sreg = src.value
-        _alu_reg.treg = 0
+        _alu_reg.treg = 1  # XXX undocumented, this is the value binutils-esp32 uses
```

Comment:
guess this is also something for the docs / for binutils.

Reply:
@igrr ^^^

Reply:
Looks like some uninitialized/oddly initialized variable in binutils. The hardware doesn't seem to care about this field for the move instruction.

Reply:
will have a look in binutils.

Reply:
did you find something? guess this is the last todo for this PR, then it could be merged?